spec/Overview.html

```diff
@@ -1324,12 +1324,30 @@ <h3>Data Types</h3>
           </dl>
         </section>
 
+        <section id="subtlecrypto-interface-tasksource">
+          <h3>Task Source</h3>
+          <dl>
+            <dt id="dfn-crypto-task-source">The <dfn>crypto task source</dfn></dt>
+            <dd>
+              This [= task source =] is used to queue tasks to resolve
+              or reject promises created in response to calls to methods
+              of {{SubtleCrypto}}.
+            </dd>
+          </dl>
+          <div class=note>
+            This specification makes no specific requirements on the
+            ordering of responses to calls to methods of {{SubtleCrypto}},
+            neither between multiple calls, nor between calls and tasks
+            from other [= task source | task sources =].
+            This task source is merely used to [= queue a task =] to
+            resolve or reject the relevant promise whenever the
+            cryptographic operation is completed, in order to
+            <a href="https://html.spec.whatwg.org/multipage/webappapis.html#event-loop-for-spec-authors">prevent race conditions</a>.
+          </div>
```

Comment:
This contradicts the usage of the task source a bit. I would leave this out.

Reply:
Which part do you think is contradictory?

The point of what I tried to write is that while the task source imposes an ordering on the queued tasks (which resolve or reject the promises), this still doesn't impose any specific requirements on the order in which the promises are resolved, because the cryptographic operations (which happen _before_ the tasks are queued) are in principle allowed to take any length of time (ignoring constant-time considerations and so on).

In other words, despite there being a queue here, it's still not guaranteed that in `[crypto.subtle.digest(..), crypto.subtle.digest(..)]` the two promises resolve in order. I think it's worth it to make that clear, but perhaps it could be phrased more clearly :s

Reply:
You're right. Feel free to disregard.